### PR TITLE
Docs: Add configurations & rest documentation

### PR DESCRIPTION
**Pull request by corneliouzbett**
Attempt to provide some documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
 Openmrs-module-queue
 ==========================
 
+> Outpatient queue management
+    
+   > https://talk.openmrs.org/t/backend-support-for-service-delivery-queues-workflows/35247
+## Prerequisites
+  - Openmrs platform version > 2.3.x
```
Comment (dkayiwa) [inline README.md:8]: Is it `>=`? 😊

Comment (corneliouzbett) [reply README.md:8]: Yes, that's correct

**Comment by ibacher** [comment]
@corneliouzbett Are you able to tweak this to reflect the status? This would be very useful to have...

**Comment by corneliouzbett** [comment]
Odd! can merge the two PRs.
All this included #8

**Pull request closed without merging**